src/plugins/loopVideos.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2023 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import { definePluginSettings } from "@api/Settings";
+import ErrorBoundary from "@components/ErrorBoundary";
+import { Devs } from "@utils/constants";
+import definePlugin, { OptionType } from "@utils/types";
+import { Tooltip, useEffect, useRef, useState } from "@webpack/common";
+
+
+const settings = definePluginSettings({
+    loop: {
+        description: "Whether to enable looping by default or not",
+        type: OptionType.BOOLEAN,
+        default: true,
+        restartNeeded: false
+    }
+});
+
+export default definePlugin({
+    name: "LoopVideos",
+    description: "Adds a loop button next to the volume slider for embedded videos",
+    authors: [Devs.Lumap],
+    settings,
+
+    patches: [
+        {
+            find: "().videoControls:",
```

Comment:
generally you should try to avoid using (). for classes. we did in the past and they changed to non () a bunch

see if this is still unique
```suggestion
            find: ".videoControls:",
```